src/pages/index.tsx

```diff
@@ -60,6 +60,9 @@ const HomePage: NextPageWithLayout = () => {
             logOut,
             targetProps: router.query,
             targetComponent: components.default,
+            tosDomainName: process.env.NEXT_PUBLIC_TOS_SUBDOMAIN_NAME,
+            termsCid: process.env.NEXT_PUBLIC_TERMS_CID,
+            privacyCid: process.env.NEXT_PUBLIC_PRIVACY_CID,
```

Comment:
Is it worth to store the endpoint to fetch terms and privacy in our .env file?

Reply:
I think the `env` variables probably make sense. I wonder if it might make things simpler if you just have two variables?

- `tosIpfsUrl` => `NEXT_PUBLIC_TOS_IPFS_URL`
- `privacyIpfsUrl` => `NEXT_PUBLIC_PRIVACY_IPFS_URL`

Or if you want to keep all 3 like you have it, maybe we rename `tosDomainName` to make it more generic?

-`ipfsHostname` => `process.env.NEXT_PUBLIC_IPFS_HOSTNAME`

Reply:
Also, don't forget to update all of the other places that use `<ComponentWrapperPage src={components.wrapper} ... />` :)

Reply:
@shelegdmitriy The new changes look good 👍